README.adoc

```diff
@@ -57,8 +55,6 @@ include::{asciidoctor-source}/kubernetes.adoc[]
 
 include::{asciidoctor-source}/fabric8.adoc[]
 
-include::{asciidoctor-source}/forge.adoc[]
```

Comment:
Here - IIUC -  we are removing `forge.adoc` but shouldn't it be `fabric8.adoc` ?

Reply:
@fabiobrz  Removing the forge.adoc is for this deprecated item:

- JBoss forge addon: remove the doc about jboss forge(https://arquillian.org/arquillian-cube/#_jboss_forge_arquillian_addon)

The fabric8.adoc should be updated to jkube. I haven't looked at the update. I created an issue for this task : https://github.com/arquillian/arquillian-cube/issues/1312

Reply:
> The fabric8.adoc should be updated to jkube. 

+1, we discussed this recently. My question is indeed about the removal, since the subject of this very PR is about boot2docker.
WDYT?

Reply:
See my latest review comments, resolving.